### DISCUSSION TITLE
How do I keep a job busy

### DISCUSSION THREAD

**Discussion by orgalaf** [taskforcesh/bullmq#368] [Q&A]
This may be an anti pattern but: I have a use case where I am trying to use bull for resource locking and management. So basically I would like to keep a job in its processing state and prevent it from stalling unless the process holding it really dies. Then, I would like it to eventually return to waiting so some other process can pick it up. Any way to do that?
Thanks!

**Comment by manast** [comment]
So as long as the job does not complete it will be kept in the active state, it will not stall as long as the CPU of the process is not at 100% since the worker class will keep renewing the job lock.
